draft-ietf-rtcweb-jsep.xml

```diff
@@ -827,8 +827,7 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
         remote endpoint will only send a single encoding per m=
         section. In addition, when the JSEP endpoint is the answerer,
         the permitted encodings for the RTPSender must be consistent
-        with the offer, but this information is currently not surfaced
-        through any API. This means that established simulcast streams
+        with the offer. This means that established simulcast streams
```

Comment:
Specifically, the section starts "JSEP currently does not provide an API to configure receipt of
   simulcast. " That's true, and sufficient basis for the rest of the text.

If you want to improve the text further, you can change

   This means that established
   simulcast streams will continue to work through a received re-offer,
   but setting up initial simulcast by way of a received offer requires
   out-of-band signaling or SDP inspection.

into

   This means that the UA can ensure that established
   simulcast streams will continue to work through a received re-offer,
   but setting up initial simulcast by way of a received offer won't work.

Reiterating: I'm happy to see it not work.